Formula/storj-uplink.rb

```diff
@@ -1,13 +1,13 @@
 class StorjUplink < Formula
   desc "Uplink CLI for the Storj network"
   homepage "https://storj.io"
-  url "https://github.com/storj/storj/archive/refs/tags/v1.57.2.tar.gz"
-  sha256 "ba0cd9d2433eaff571082ef52bbc26ea74ef3b90760919cb48176dac3936f326"
+  url "https://github.com/storj/storj/archive/refs/tags/v1.63.1.tar.gz"
+  sha256 "3d175422e2bc371afcd6d6f591c96ebc8f780c8ce2fcbf20ed2d50a3e48bc8c7"
   license "AGPL-3.0-only"
 
   livecheck do
     url :stable
-    regex(/^v?(\d+(?:\.\d+)+)$/i)
+    strategy :github_latest
```

Comment:
Upstream release flow looks like they tag a version as pre-release before promoting it to GitHub latest, e.g. previous livecheck detected [1.64.1](https://github.com/storj/storj/releases/tag/v1.64.1) is currently a pre-release.

Looks like this happened for `1.63.1` given previous attempt #109654